CHANGELOG.md

```diff
@@ -1,108 +1,91 @@
+| Bug   | Wishlist page error when configured variant added                                                                                                 | [3703][]                     |
+| Bug   | Fix Cypress Page Builder tests                                                                                                                    | [3715][]                      |
+| Bug   | Dynamic Blocks GraphQL returns non-qualified blocks                                                                                        | [3678][], [3][]               |
+| Bug   | Remove virtual products extension                                                                                                                 | [3612][]                      |
+| Bug   | Possible wrong usage of useMemo                                                                                                        | [3660][]                      |
+| Bug   |  PWA multiple wish lists: storefront still showing create wishlist button when reaching the max number of allowed wish lists configured in Commerce    | [3668][]                      |
+| Bug   | Header - Logo dimensions overlap with main content                                                                                                | [3659][]                      |
+| Bug   | Click target for long-spanning tabs is inconsistent                                                                                          | [3676][]                      |
+| Bug   | Venia sample data personalized content not updating                                                                                               | [20][]                        |
 
 ## Documentation updates
 
--  **contactUs Mutation**: Added the [contactUs][] mutation topic to the GraphQL Developer Guide.
+-   Added [best practices documentation](https://developer.adobe.com/dreamweaver/guides/best-practices/index.html) for Customer Segments.
 
 ## Known Issues
 
--  Safari (macOS version) does not show toast messages or indicators when Venia switches between online and offline. This is an issue with Safari, not Venia. Safari always reports `true` for `navigator.onLine` — even when offline. We have submitted this issue to Apple. If you have an Apple account, you can search for the issue using this Feedback ID: FB9802994.
+-  None!
 
 ## Upgrading from a previous version
 
-Use the steps outlined in this section to update your [scaffolded project][] from 12.1.0 to 12.2.0.
+Use the steps outlined in this section to update your [scaffolded project][] from 12.2.0 to 12.3.0.
 See [Upgrading versions][] for more information about upgrading between PWA Studio versions.
 
 [scaffolded project]: https://magento.github.io/pwa-studio/tutorials/pwa-studio-fundamentals/project-setup/
```

Comment:
Should update these links to point to the new site

Reply:
Good point. I thought I updated those numbers.